cdap-client/src/main/java/co/cask/cdap/client/util/RESTClient.java

```diff
@@ -19,12 +19,12 @@
 import co.cask.cdap.client.config.ClientConfig;
 import co.cask.cdap.client.exception.DisconnectedException;
 import co.cask.cdap.common.UnauthenticatedException;
+import co.cask.cdap.common.http.HttpMethod;
```

Comment:
Any other projects that use this class (RESTClient) would have compilation failures.

Reply:
Fixed one instance in: https://github.com/caskdata/cdap-integration-tests/pull/954